scripts/reactor_deployment.py

```diff
@@ -0,0 +1,496 @@
+# These functions outline different reactor deployment algorithms,
```

Comment:
Might be worth making this a class since these functions share `df` and `ar_dict` it looks like (and simplifies doc strings so they don't have to repeat)